examples/svc_mnsit.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
Probably, need to create a separate folder for sklearnex examples, don't mix them with daal4py examples

Reply:
I think better move daal4py to separate folder https://github.com/intel/scikit-learn-intelex/pull/679
Because, when daal4py move to onedal repo, folder sklearnex will be strange look :)

Reply:
What about separating examples and Jupyter notebooks? + providing some sort of online notebooks support -via collab for example?